vars/debian.yml

```diff
@@ -24,3 +24,10 @@ keepalived_ubuntu_src: "native" #no support of ppas under debian -> forced sourc
 ## Prevent start on install
 prevent_start_file: "/etc/systemd/system/keepalived.service"
 prevent_start_file_content: false
+
+# Packages for Selinux
+keepalived_selinux_packages:
```

Comment:
that can probably be an empty list instead. I doubt someone with debian will have selinux...

Reply:
I think we need to have [] to make it an empty list, else it will be null and start to fail.